### ISSUE TITLE
hyperparameters extensions

### ISSUE THREAD

**Issue by MK8BK** [MK8BK/Deep_Learning_ISD#1]
glorot initialisation
momentum
dropout
learning rate decay
implement convolution and pooling layers
review general architecture
